src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -46,6 +46,7 @@ public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final En
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
     {
+
```

Comment:
I think you can remove the whitespace already

Reply:
Woops, on it.